Chat UI fixes

philipperolet opened the pull request:
- Sparkle components for clipboard, feedback
- proper fix of clipboard to avoid it overlaying text (float right)
- note: for clipboard, at the time used existing sparkle svg (ClipboardIconStroke, ClipboardIconSolid) rather than former  heroicons (ClipboardDocmentListIcon, ClipboardDocumentCheckIcon), cc @Duncid does it look good enough or would it be better with previous icons? in which case can you add them to sparkle :pray: ?
- note 2: linting error in tailwind.config.css => had it slipped through? or was there a reason?

Duncid commented:
Can you post a screenshot?

spolu commented on front/components/use/chat/MessageFeedback.tsx:
```diff
@@ -19,40 +24,23 @@ export function MessageFeedback({
     >
-      <div
+      <IconButton
+        type={message.feedback === "positive" ? "primary" : "secondary"}
```
Should be tertiary

spolu commented on front/components/use/chat/MessageFeedback.tsx:
```diff
@@ -19,40 +24,23 @@ export function MessageFeedback({
+        className="ml-1"
+      />
+      <IconButton
+        type={message.feedback === "negative" ? "primary" : "secondary"}
```
Should be tertiary (grey instead of black)

spolu reviewed (changes requested):
Looks much nicer!! \o/

Minor change request.

spolu commented:
THank for fixing the tailwind config. Looks very reasonable

spolu commented:
@Duncid deploying on front-edge for you

spolu commented:
Front edge is out @Duncid

Duncid commented:
That's what I see:

<img width="122" alt="image" src="https://github.com/dust-tt/dust/assets/4435185/bdda7590-6690-4fef-8d05-a27a46463fa0">

We need to use Sparkle component:
- **IconButton** for clipboard
- **IconToggleButton** for up down thumbs
- In both case **Type = "tertiary"**

Use sparkle icons "hand-thumb-up", "hand-thumb-down", "clipboard" in the MINI version

spolu commented:
Note mini version is -Icon (no solid no Stroke)

philipperolet commented:
@Duncid thx
I changed to the mini icons & switched the clipboard to an IconButton rather than IconToggle. **To note**:
- icons in mini look the same as in solid / stroke in this situation
   - did you mean for better code quality we should use them rather than stroke / solid? if yes, good it's done :)
   - if you meant that you need the icons to look smaller, then I'll investigate (this is strange because height/width was not set anywhere else)
- I used a checkcircleicon for 1s as feedback to the user that the copy has been done

**IconButton vs IconToggleButton for thumbs**
I had originally used an IconToggle for the thumbs but it didn't work because toggling one should untoggle the other, and the current IconToggle component does not allow this behaviour AFAICT.  This is why I switched to IconButton

I can change the component to allow this behaviour and then use it if you think it's best @spolu @Duncid

spolu reviewed (approved):
LGTM code wise 👍
